### ISSUE TITLE
[DD] Configure the upload of source maps for FE apps

### ISSUE THREAD

**Issue by jristau1984** [edx/edx-arch-experiments#666]

**Comment by jristau1984** [comment]
@abdullahwaheed here is the placeholder... let me know if you want me to add details.

**Comment by adamstankiewicz** [comment]
[inform] Here's a couple related PRs (should ready for a review) for how we might upload JS source maps to Datadog during MFE deployment:
* https://github.com/edx/tubular/pull/3
* https://github.com/edx/edx-internal/pull/10993

cc @abdullahwaheed @jristau1984

**Comment by adamstankiewicz** [comment]
This work was completed, and recently rolled out for stage/production/edge MFEs.

Example successful output for production deployment of frontend-app-learning: 

```shell
Starting upload with concurrency 20. 
Will look for sourcemaps in target/dist
Will match JS files for errors on files starting with /
version: 072d608c64767b9997f0489f0862698714d690d2 service: edx-frontend-app-learning project path: frontend-app-learning/
Uploading sourcemap target/dist/106.474fa2bbc3e6fa7afd99.js.map for JS file available at 106.474fa2bbc3e6fa7afd99.js
Uploading sourcemap target/dist/210.f533b57b98d2d5584268.js.map for JS file available at 210.f533b57b98d2d5584268.js
Uploading sourcemap target/dist/28.5fa305c727a27c5f73c4.js.map for JS file available at 28.5fa305c727a27c5f73c4.js
Uploading sourcemap target/dist/35.78fa71b6ffaf0a073d45.js.map for JS file available at 35.78fa71b6ffaf0a073d45.js
Uploading sourcemap target/dist/437.57a1600a235abbad412a.js.map for JS file available at 437.57a1600a235abbad412a.js
Uploading sourcemap target/dist/443.9b1f0b2197e15128928f.js.map for JS file available at 443.9b1f0b2197e15128928f.js
Uploading sourcemap target/dist/680.181a30c64111587fc73c.js.map for JS file available at 680.181a30c64111587fc73c.js
Uploading sourcemap target/dist/700.0f22a50f4c0146e25d49.js.map for JS file available at 700.0f22a50f4c0146e25d49.js
Uploading sourcemap target/dist/715.fd3aa624e255dc0538ef.js.map for JS file available at 715.fd3aa624e255dc0538ef.js
Uploading sourcemap target/dist/724.25aa06ea2b6b272a1efe.js.map for JS file available at 724.25aa06ea2b6b272a1efe.js
Uploading sourcemap target/dist/901.202fb5acec0d67427a89.js.map for JS file available at 901.202fb5acec0d67427a89.js
Uploading sourcemap target/dist/974.558db8619526b7b79d0f.js.map for JS file available at 974.558db8619526b7b79d0f.js
Uploading sourcemap target/dist/app.86d61acc2e76a742d942.js.map for JS file available at app.86d61acc2e76a742d942.js
Uploading sourcemap target/dist/runtime.1d1b13820e1f1c45d668.js.map for JS file available at runtime.1d1b13820e1f1c45d668.js

Command summary:
✅ Uploaded 14 sourcemaps in 1.158 seconds.
```

**Comment by adamstankiewicz** [comment]
If MFEs have any issues with failing GoCD pipelines, etc. for MFEs, please reach out! Thanks.
